### PR TITLE
Update com_dataviewer download code

### PR DESCRIPTION
**Pull request by anthonyfuentes**
Removing output buffer functions allows CSV data to be sent to client

fixes: https://purr.purdue.edu/support/ticket/1962

### CODE REVIEW

File: core/components/com_dataviewer/site/filter/csv.php [hunk 1]
```diff
@@ -57,7 +55,6 @@ function filter($res, &$dd, $ob_mode = false)
 		}
 	}
 	$csv .= $nl;
-	$csv .= columnsMetadata($dd);
```
Comment (zooley) [inline core/components/com_dataviewer/site/filter/csv.php]: Why was this line removed?

Comment (anthonyfuentes) [reply core/components/com_dataviewer/site/filter/csv.php]: The metadata that was added to the columns didn't seem to be anything that the user would be interested in. Without the metadata, the CSV more closely resembles the appearance of the data as seen in the dataviewer.
